packages/p2p/src/components/my-ads/my-ads.scss

```diff
@@ -250,9 +250,9 @@
         &-popovers {
             background-color: var(--general-main-1);
             display: flex;
-            height: 100%;
+            height: 99%;
```

Comment:
Why 99%?

Reply:
> Why 99%?

because popover is overlapping row bottom border, 99% is fixing it